common/minimodel/__init__.py

```diff
@@ -1008,3 +1013,52 @@ def total_used(self) -> timedelta:
 
     def observations(self) -> List[Observation]:
         return self.root_group.observations()
+
+
+class Plans:
+    """
+    A collection of Nightly Plan from a corresponding Site
+    """
+    def __init__(self, night_events):
```

Comment:
I'm really pushing for us to include type hints to make this code more production-worthy so that warnings will be generated if we pass incorrect data types in, which is far too easy to do in Python.

```
def __init__(self, night_events: NightEvents):
```